include/swift/Basic/BasicBridging.h

```diff
@@ -512,6 +512,18 @@ struct BridgedVersionTuple {
   llvm::VersionTuple unbridged() const;
 };
 
+//===----------------------------------------------------------------------===//
+// MARK: BridgedSwiftClosure
+//===----------------------------------------------------------------------===//
+
+/// Wrapping a pointer to a Swift closure `(UnsafeRawPointer?) -> Void`
+/// See 'withBridgedSwiftClosure(closure:call:)' in ASTGen.
+struct BridgedSwiftClosure {
+  const void *_Nonnull closure;
+
+  BRIDGED_INLINE void operator()(const void *_Nonnull);
```

Comment:
```suggestion
  BRIDGED_INLINE void operator()(const void *_Nullable);
```

Currently we only use non-null values, but `_Nullable` so that other future bridging functions can call the callback with a null argument.